views/login.handlebars

```diff
@@ -0,0 +1 @@
+<button >CLICK TO LOGIN/SIGNUP</button>
```

Comment:
how about wrapping this in an `<a>` tag?

`<a href="/login"><button>CLICK TO LOGIN/SIGNUP</button></a>`